declarations/eharmony.json

```diff
@@ -0,0 +1,13 @@
+{
+  "name": "eharmony",
+  "documents": {
+    "Terms of Service": {
+      "fetch": "https://www.eharmony.co.uk/termsandconditions/",
+      "select": ".legalTextBox"

```

Comment:
Unfortunalty i didn't find an easy way to delete the model cancellation form and the end of the terms of service.

Reply:
I removed the links to the pdf file, fixed in [﻿﻿9a0a24baba91f739bba346e273ee41335c289314](https://github.com/OpenTermsArchive/services-dating/pull/2/commits/9a0a24baba91f739bba346e273ee41335c289314). I admit this is not the best way to do so, but it's working.